testprojects/src/python/interpreter_selection/resolver_blacklist_testing/BUILD

```diff
@@ -16,3 +16,19 @@ python_requirement_library(
     python_requirement('jupyter'),
   ]
 )
+
+# Test non-blacklisted backport usage.
+python_binary(
+  name='test_py2',
```

Comment:
maybe add compatibility to <3 to be specific and remain runnable if/when we switch default to Python3?

Reply:
Sure thing.